test/suite/test_import10.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-2021 MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import wiredtiger
+from wtscenario import make_scenarios
+from test_import01 import test_import_base
+
+# test_import10.py
+#    Run live import/export while backup cursor is open
+class test_import10(test_import_base):
+    uri = 'file:test_import10.wt'
+    create_config = 'allocation_size=512,key_format=i,value_format=i'
+    scenarios = make_scenarios([
+        ('file_metadata', dict(repair=False)),
+        ('repair', dict(repair=True)),
+    ])
+
+    def test_backup_with_live_import(self):
+        """
+        Test live import in a 'wt' process while backup cursor is open.
+        """
+        # Create and populate the table.
+        self.session.create(self.uri, self.create_config)
+        cursor = self.session.open_cursor(self.uri)
+        for i in range(1, 1000):
+            cursor[i] = i
+        cursor.close()
+        self.session.checkpoint()
+
+        # Export the metadata for the file.
+        c = self.session.open_cursor('metadata:', None, None)
+        original_db_file_config = c[self.uri]
+        c.close()
+
+        self.session.drop(self.uri, 'remove_files=false')
+
+        # Can't open the cursor on the file anymore since we dropped it.
+        self.assertRaisesException(wiredtiger.WiredTigerError,
+            lambda: self.session.open_cursor(self.uri))
+
+        # Open backup cursor
+        bkup_c = self.session.open_cursor('backup:', None, None)
+
+        # Import file with the associated file metadata AND no metadata, wish to repair it.
```

Comment:
This comment doesn't exactly make sense as written. Consider rewording as "First construct the config string for the default or repair import scenario, then call create to import the table."

Reply:
done!